src/orquestra/sdk/secrets/_api.py

```diff
@@ -11,16 +11,25 @@
 from . import _auth, _exceptions, _models
 
 
+def _translate_to_zri(workspace_id: str, secret_name: str) -> str:
+    """
+    Create ZRI from workspace_id and secret_name
+    """
+    return f"zri:v1:unused:0:{workspace_id}:secret:{secret_name}"
+
+
 def get(
     name: str,
     *,
+    workspace_id: t.Optional[str] = None,
     config_name: t.Optional[str] = None,
 ) -> str:
     """
     Retrieves secret value from the remote vault.
 
     Args:
         name: secret identifier.
+        workspace_id: ID of the workspace. Using platform-defined default if omitted
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can you explain what the default is? The docstring isn't helpful for folks who don't know the details of the service.

same for the other instances of this

Reply:
I wanted to abstain from that, as it might change in the future and doc might easily get outdated, but ill put what is current default 👍